tests/functional/modules/test_zos_backup_restore.py

```diff
@@ -795,3 +821,40 @@ def test_backup_into_gds(ansible_zos_module, dstype):
     finally:
         hosts.all.shell(cmd=f"drm ANSIBLE.* ")
 
+
+def test_backup_tolerate_enqueue(ansible_zos_module):
+    hosts = ansible_zos_module
+    default_data_set_name_1 =  get_tmp_ds_name()
+    default_data_set_name_2 =  get_tmp_ds_name()
+    temp_file = f"/tmp/MEM2"
+    data_sets_hlq = "ANSIBLE.**"
+    data_sets_backup_location = get_tmp_ds_name()
+    try:
+        hosts.all.shell(cmd="dtouch {0}".format(default_data_set_name_1))
+        hosts.all.shell(cmd="dtouch {0}".format(default_data_set_name_2))
+        hosts.all.shell(cmd="""decho "HELLO WORLD" "{0}" """.format(default_data_set_name_1))
+        hosts.all.shell(cmd="""decho "HELLO WORLD" "{0}" """.format(default_data_set_name_2))
+        hosts.all.file(path="/tmp/disp_shr/", state="directory")
```

Comment:
Can you use temporary directories for testing? Since we are trying to improve portability and parallel execution this will make it easier.